corral.json

```diff
@@ -0,0 +1,13 @@
+{
+  "packages": [
+    "immutable-json"
+  ],
+  "deps": [],
+  "info": {
+    "description": "like the ponylang stdlib json library - just everything `val`",
+    "homepage": "https://github.com/mfelsche/pony-immutable-json",
+    "license": "BSD-2-Clause",
```

Comment:
I'd prefer Apache 2.0 tbh

Reply:
I left it as a placeholder, use the one you prefer :)